src/phantom.js

```diff
@@ -82,7 +82,7 @@ function ResourceHandler(htmlroot, isWindows, resolve) {
             }
             networkRequest.changeUrl(url);
         } else if (ignoreRequests.test(url)) {
-            networkRequest.abort();
+            networkRequest.cancel();
```

Comment:
I wish I could figure this out once and for all but I'm completely stuck. I'm not familiar with phantom.js :/